packages/okta-vue/test/e2e/harness/src/App.vue

```diff
@@ -2,7 +2,7 @@
   <div id="app">
     <router-link to="/" tag="button" id='home-button'> Home </router-link>
     <button v-if='authenticated' v-on:click='logout' id='logout-button'> Logout </button>
-    <button v-else v-on:click='$auth.loginRedirect' id='login-button'> Login </button>
+    <button v-else v-on:click='$auth.loginRedirect("/")' id='login-button'> Login </button>
```

Comment:
This is the primary reason for the breaking change.

Fun fact - If no params are passed in, Vue will set the first parameter as a `MouseEvent` object.

Reply:
o_0